apps/mobile/src/features/approver/components/approver-account-card.tsx

```diff
@@ -0,0 +1,27 @@
+import { AvatarIcon } from '@/components/avatar-icon';
+import { AccountListItem } from '@/features/account-list/account-list-item';
+import { useWallets } from '@/store/wallets/wallets.read';
+import { t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import { Text, Theme } from '@leather.io/ui/native';
+
+export function ApproverAccountCard() {
+  const theme = useTheme<Theme>();
+  const { list: walletsList } = useWallets();
+  const wallet = walletsList[0];
+  /* eslint-disable-next-line lingui/no-unlocalized-strings  */
+  const testAccountName = 'testName';
+  return (
+    <>
+      <Text variant="label01">{t`With account`}</Text>
+      <AccountListItem
+        accountName={testAccountName}
+        address={t`Address`}
+        balance={t`$1234`}
```

Comment:
I made an `AccountBalance` and `AccountAddress` component in my PR that can go here at some point.